<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewPocketPC</class>
 <widget class="QDialog" name="AddNewPocketPC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="ModelLabel">
       <property name="text">
        <string>Модель</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ModelLineEdit">
       <property name="text">
        <string comment="Модель"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="CompanyLabel">
       <property name="text">
        <string>Фирма</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="CompanyLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="ProcessorLabel">
       <property name="text">
        <string>Процессор</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ProcessorLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="RAM_ROMLabel">
       <property name="text">
        <string>ОЗУ/ПЗУ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="RAM_ROMLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="ScreenResolutionLabel">
       <property name="text">
        <string>Разрешение экрана</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ScreenResolutionLineEdit"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddNewPocketPC</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddNewPocketPC</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
